src/main/java/com/qaprosoft/carina/demo/gui/pages/localizationSample/MainPage.java

```diff
@@ -24,7 +24,7 @@
 
 public class MainPage extends AbstractPage {
 
-    @FindBy(xpath = "//div[@id='js-lang-lists']/div[1]/ul/li/a")
+    @FindBy(xpath = "//div[@id='js-lang-lists']//a")
```

Comment:
Lets use just the id, if it is possible

Reply:
I don't know if it's possible because I want to get the list of links inside the div